tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/colorization_evaluator.py

```diff
@@ -54,6 +55,14 @@ def from_configs(cls, config, delayed_model_loading=False):
             launcher_settings['device'] = 'CPU'
         launcher = create_launcher(launcher_settings, delayed_model_loading=True)
         network_info = config.get('network_info', {})
+        colorization_network = network_info.get('colorization_network', {})
+        verification_network = network_info.get('verification_network', {})
+        colorization_network['net_type'] = 'colorization_network'
+        verification_network['net_type'] = 'verification_network'
```

Comment:
why do you need this? they have this info as keys in network info, why is it not enough for you?